### PR TITLE
Add business rules to the create feed readme

### PR DESCRIPTION
**Pull request by wesleyalforddot**
This PR adds current business rules to the create feed markdown. This is guidance for how to implement the specification to ensure standardization from data produces and to facilitate data consumer usage.

### CODE REVIEW

**Review by j-d-b — CHANGES_REQUESTED** [review]
I'd highly recommend renaming all the png images to more clearly represent what they are. Out of context, `scenario1.png` has no meaning. I think you could just change them to what the header name is in `example-work-zone-scenarios.md` file. That is, `scenario1.png` renamed to `simple-work-zone-scenario.png`.

File: create-feed/README.md [hunk 1]
```diff
@@ -45,6 +45,23 @@ The [schemas](/create-feed/schemas) directory includes a JSON Schema for the fol
...
+
+1. Each direction of travel receives a separate road event. For example, a work zone on a two lane road, with opposite travel directions ↑↓, are two separate road events.
+2. Construction requiring alternating traffic flows in a lane should have a road event for both directions. Additionally, roadways that during normal operation alternate the flow of traffic based on time of day should have separate road events that have corresponding start and end times.
+3. The preferred geometry type representation of a road_event is LineString. In the event that coordinates between the beginning and ending points is an unavailable, the road event will be specified by a MultiPoint geometry.  
+4. A GeoJSON feed will be comprised of a single object of the road_event_feed_info type and feature properties will be of type FeatureCollection. 
+5. The features array represents a list of one or more road events. Each object in this array has three properties: type, properties, and geometry. Type will always be feature. Properties will provide all details of the road event except for geometry and geometry type, which are specified by the geometry property.
```
Comment (j-d-b) [inline create-feed/README.md:55]: Again this is a helpful text description of what the schema says but it is not a business rule, it is required by the schema.

File: create-feed/README.md [hunk 2]
```diff
@@ -45,6 +45,23 @@ The [schemas](/create-feed/schemas) directory includes a JSON Schema for the fol
...
+6. A mobile work zone should consider the update frequency of the WZDx data feed found in the metadata datafeed_frequency_update field. A mobile work zone should define the start and end of the work zone to be based on the approximate distance covered in the time between elapsed feed update cycles.
```
Comment (j-d-b) [inline create-feed/README.md:56]: This needs to be updated for v3.0 (metadata is gone)

File: create-feed/README.md [hunk 3]
```diff
@@ -45,6 +45,23 @@ The [schemas](/create-feed/schemas) directory includes a JSON Schema for the fol
...
+7. A cascading multilane closure should consider the speed of vehicles traveling through the work zone. If the distance between lane closure would not allow for a significant period of travel in a to be closed lane, then the lane should be considered closed to avoid traffic delays and potential crashes.
```
Comment (j-d-b) [inline create-feed/README.md:57]: Where did this come from? Can you link an issue or discussion?

Comment (wesleyalforddot) [reply create-feed/README.md]: This was made in reference to scenario 6 from the annotated examples here [https://github.com/usdot-jpo-ode/jpo-wzdx/discussions/131](https://github.com/usdot-jpo-ode/jpo-wzdx/discussions/131). If you have suggested wording or don't think this rules is reasonable let me know and I can remove it.

**Comment by j-d-b** [comment]
@wesleyalforddot this was never approved. Please wait until you have approved reviewers before merging PRs. All PRs should have at least one approved review from a maintainer/co-chair, as per the charter.

**Pull request merged**